hack/make-rules/verify/all.sh

```diff
@@ -25,49 +25,59 @@ res=0
 
 # run all verify scripts, optionally skipping any of them
 
+FAILED=()
 if [[ "${VERIFY_GO_LINT:-true}" == "true" ]]; then
-  echo "verifying go lints ..."
-  hack/make-rules/verify/golangci-lint.sh || res=1
+  name="go lints"
+  echo "verifying $name ..."
+  hack/make-rules/verify/golangci-lint.sh || { res=1; FAILED+=($name); echo "ERROR: $name failed"; }
   cd "${REPO_ROOT}"
 fi
 if [[ "${VERIFY_GOFMT:-true}" == "true" ]]; then
-  echo "verifying go fmt ..."
-  hack/make-rules/verify/gofmt.sh || res=1
+  name="go fmt"
+  echo "verifying $name"
+  hack/make-rules/verify/gofmt.sh || { res=1; FAILED+=($name); echo "ERROR: $name failed"; }
   cd "${REPO_ROOT}"
 fi
 if [[ "${VERIFY_FILE_PERMS:-true}" == "true" ]]; then
-  echo "verifying .sh files permissions ..."
-  hack/make-rules/verify/file-perms.sh || res=1
+  name=".sh files permissions"
+  echo "verifying $name"
+  hack/make-rules/verify/file-perms.sh || { res=1; FAILED+=($name); echo "ERROR: $name failed"; }
   cd "${REPO_ROOT}"
 fi
 if [[ "${VERIFY_SPELLING:-true}" == "true" ]]; then
-  echo "verifying spelling ..."
-  hack/make-rules/verify/misspell.sh || res=1
+  name="spelling"
+  echo "verifying $name"
+  hack/make-rules/verify/misspell.sh || { res=1; FAILED+=($name); echo "ERROR: $name failed"; }
   cd "${REPO_ROOT}"
 fi
 if [[ "${VERIFY_LABELS:-true}" == "true" ]]; then
-  echo "verifying labels ..."
-  hack/make-rules/verify/labels.sh || res=1
+  name="labels"
+  echo "verifying $name"
+  hack/make-rules/verify/labels.sh || { res=1; FAILED+=($name); echo "ERROR: $name failed"; }
   cd "${REPO_ROOT}"
 fi
 if [[ "${VERIFY_CODEGEN:-true}" == "true" ]]; then
-  echo "verifying codegen ..."
-  hack/make-rules/verify/codegen.sh || res=1
+  name="codegen"
+  echo "verifying $name"
+  hack/make-rules/verify/codegen.sh || { res=1; FAILED+=($name); echo "ERROR: $name failed"; }
   cd "${REPO_ROOT}"
 fi
 if [[ "${VERIFY_TSLINT:-true}" == "true" ]]; then
-  echo "verifying tslint ..."
-  hack/make-rules/verify/tslint.sh || res=1
+  name="tslint"
+  echo "verifying $name"
+  hack/make-rules/verify/tslint.sh || { res=1; FAILED+=($name); echo "ERROR: $name failed"; }
   cd "${REPO_ROOT}"
 fi
 if [[ "${VERIFY_PYLINT:-true}" == "true" ]]; then
-  echo "verifying pylint ..."
-  hack/make-rules/verify/pylint.sh || res=1
+  name="pylint"
+  echo "verifying $name"
+  hack/make-rules/verify/pylint.sh || { res=1; FAILED+=($name); echo "ERROR: $name failed"; }
   cd "${REPO_ROOT}"
 fi
 if [[ "${VERIFY_BOILERPLATE:-true}" == "true" ]]; then
-  echo "verifying boilerplate ..."
-  hack/make-rules/verify/boilerplate.sh || res=1
+  name="boilerplate"
+  echo "verifying $name"
+  hack/make-rules/verify/boilerplate.sh || { res=1; FAILED+=($name); echo "ERROR: $name failed"; }
```

Comment:
Nit: you could drop the `res=1` and just check if `FAILED` array is not empty ("if res is not 0 or if FAILED is not empty, show errors...").

Reply:
done